app/models/commit.rb

```diff
@@ -0,0 +1,10 @@
+class Commit < ActiveRecord::Base
+  attr_accessible :commit_oid, :repository
+  attr_accessible :author, :author_date
+  attr_accessible :commit_date, :committer
```

Comment:
`attr_accessible :committer, :commit_date` for consistency in the ordering with the author-line.